Common/Helpers/AzureTableStorageClient.cs

```diff
@@ -10,37 +11,63 @@
 
 namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Helpers
 {
-    public class AzureTableStorageHelper : IAzureTableStorageHelper
+    public class AzureTableStorageClient : IAzureTableStorageClient
     {
-        private readonly string _storageConnectionString;
-        private readonly string _tableName;
         private CloudTable _table;
+        private readonly CloudTableClient _tableClient;
+        private readonly string _tableName;
 
-        public AzureTableStorageHelper(string storageConnectionString, string tableName)
+        public AzureTableStorageClient(string storageConnectionString, string tableName)
         {
-            _storageConnectionString = storageConnectionString;
+            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(storageConnectionString);
+            _tableClient = storageAccount.CreateCloudTableClient();
```

Comment:
let's start using "this._tableClient" to differentiate between local variables and class variables.